### PR TITLE
API-21515-Pingdom Probes

### PR DESCRIPTION
**Pull request by jkelly-va**

### CODE REVIEW

File: integration-ids.conf [hunk 1]
```diff
@@ -17,6 +17,7 @@ export SLACK_API_ALARMS_CLINICAL_FHIR=113879
 export SLACK_API_ALARMS_COMMUNITY_CARE=107294
 export SLACK_API_ALARMS_DATA_QUERY=106657
 export SLACK_API_ALARMS_DATAMART=124302
+export SLACK_API_ALARMS_DIRECT_DEPOSIT_MANAGEMENT=000000
```
Comment (mramirez-va) [inline integration-ids.conf:20]: This will need to be in a separate PR once we have the probe in Pingdom

File: user-ids.conf [hunk 2]
```diff
@@ -37,6 +37,7 @@ export STATUSPAGE_SANDBOX_BENEFITS_REFERENCE_DATA_V1=14711916
 export STATUSPAGE_SANDBOX_CLINICAL_FHIR=14642203
 export STATUSPAGE_SANDBOX_COMMUNITY_CARE_ELIGIBILITY=14491285
 export STATUSPAGE_SANDBOX_DECISION_REVIEWS=14533791
+export STATUSPAGE_SANDBOX_DIRECT_DEPOSIT_MANAGEMENT=00000000
```
Comment (mramirez-va) [inline user-ids.conf:40]: This will need to be in a separate PR once we have the probe in Pingdom

**Review by mramirez-va — CHANGES_REQUESTED** [review]

**Review by mramirez-va — DISMISSED** [review]
remove statuspage integration place holders

File: pingdom-checks/direct-deposit-management.ping [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# Direct Deposit Management Health Check
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-direct-deposit-management-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/direct-deposit-management/v1/healthcheck/liveness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="direct-deposit-management" \
```
Comment (mramirez-va) [inline pingdom-checks/direct-deposit-management.ping:19]: @jkelly-va fyi, I renamed this from `direct-deposit` to match the full API name.

**Review by evelez-va — APPROVED** [review]

**Review by mramirez-va — APPROVED** [review]

**Review by jhulbert-va — APPROVED** [review]
LGTM

**Pull request merged**